integration-tests/src/testnetSetup.ts

```diff
@@ -1,11 +1,41 @@
+import { providers, Wallet } from 'ethers';
 import { default as globalSetup } from './globalSetup';
 
+const createWallet = (who: string) => {
+    const lenderPrivateKey = process.env.LENDER_PRIVATE_KEY;
+    if (lenderPrivateKey === undefined) {
```

Comment:
```suggestion
    if (!lenderPrivateKey) {
```

Reply:
I will not apply this suggestion. There are indeed some settings which need to remain `false`. `undefined` is a 3rd state meaning value is not defined! I will add a comment in the code though, to make it more clear.

Reply:
if `lenderPrivateKey` equals false, null, 0 or "" you don't want it to throw an error? @atodorov